files/ko/learn/javascript/first_steps/arrays/index.html

```diff
@@ -634,22 +631,26 @@ <h6 id="Playable_code_2">Playable code 2</h6>
 
 <p>{{ EmbedLiveSample('Playable_code_2', '100%', 700, "", "", "hide-codepen-jsfiddle") }}</p>
 
+<h2 id="Test_your_skills!">당신의 기량을 시험해 보세요!</h2>
+
+<p>당신은 이 문서의 끝에 도달했지만, 가장 중요한 정보들을 기억할 수 있으신가요? 당신은 나아가기 전에 이 정보들을 보유하고 있다는 것을 확인하는 추가적인 테스트들을 찾을 수 있습니다 — 다음을 보세요: <a href="/en-US/docs/Learn/JavaScript/First_steps/Test_your_skills:_Arrays">Test your skills: Arrays</a></p>
+
 <h2 id="결론">결론</h2>
 
-<p>위에 글 읽어보니, 배열이 꽤 유용해 보인다는거 알꺼임; JavaScript에서 배열은 겁나 많이 쓰인다. 배열의 모든 항목 마다 똑같은 작업을 수행하려고 루프(loop)를 돌리니까 같이 알아놓으면 개꿀. 다음 모듈(챕터)에서 루프(loop)에 관한 기초부터 알려줄꺼니까 쫄지 말고 달려. 갈 길이 멀다. 이 모듈은 이제 다 봤어!</p>
+<p>이 문서를 쭉 읽은 후, 우리는 당신이 배열이 꽤 유용해 보인다는 것에 동의할 것이라고 확신합니다. 당신은 배열이 자바스크립트의 모든 곳에서 불쑥 나타나는데, 이는 종종 배열의 모든 원소에 같은 것을 하기 위해 반복문과 관련하는 것을 볼 수 있을 것입니다. 우리는 당신에게 다음 모듈에서 반복문에 대해 알기 위해 있는 모든 유용한 기본을 가르칠 것이지만, 우선 당신은 당신에게 박수를 보내고 충분한 자격이 있는 휴식을 취하세요. 당신은 이 모듈의 모든 문서를 헤쳐 나왔으니까요!</p>
```

Comment:
안녕하세요, 번역 하시느라 너무 고생 많으셨습니다. 기존에 쓰인 표현이 부적절한 것들이 있었던 거 같은데 잘 다듬어진 거 같아요!
번역 대부분 잘 읽히는데, 결론 부분은 조금 가벼운 톤으로 가보면 어떨까 싶어서 약간의 번역투가 있는 부분에 제가 말을 조금 고쳐봤습니다. 그저 suggestion일 뿐이니 불필요하다고 판단되시면 수정하지 않으셔도 괜찮지만 한번 코멘트로 남겨봅니다! 

```suggestion
<p>이 문서를 쭉 읽으셨다면 배열이 꽤 유용해 보인다는 것에 동의할 겁니다. 배열은 종종 모든 원소에 동일한 것을 하기 위해 반복문과 함께 자바스크립트의 모든 곳에서 찾아볼 수 있습니다. 다음 모듈에서 반복문에 대해 알기 위해 있는 모든 유용한 기본을 가르칠 것이지만, 우선 스스로에게 박수를 보내고 휴식을 취하세요. 이 모듈의 모든 문서를 독파하셨으니 잠깐 쉬어가도 좋습니다!</p>
```

Reply:
검수해주셔서 감사합니다.